GrafanaUI: Create Box component (#73637)
grafana/grafana: packages/grafana-ui/src/components/Layout/Box/Box.mdx

import { Meta, ArgTypes } from '@storybook/blocks';
import { Box } from './Box';

<Meta title="MDX|Box" component={Box} />

# Box

The Box Component is the most basic layout component. It can be used to build more complex components and layouts with properties
that use our design tokens instead of using CSS.

### Usage

#### When to use

Use it whenever you would use custom CSS.

#### When not to use

> If you need more specific layouts like Flex or Grid, use those specific components instead.

Reviewer: ```suggestion
If you need layout styles, use the Stack, Flex or Grid components instead.
```
I don't know, maybe this makes more sense?
